java/solomiia_tymoshchuk/src/com/zoolatech/lecture1/tasks/_4/Task4.java

```diff
@@ -0,0 +1,38 @@
+package com.zoolatech.lecture1.tasks._4;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a day of the week as an integer number (1 - Monday, 2 - Tuesday, …, 7 - Sunday) and output
+ * “Need to go to work…” if the day is a weekday or “Sleeping…”
+ * if it’s a weekend. Create two versions of the program:
+ * one should use the if-else statement, second - the conditional operator.
+ **/
+public class Task4 {
+    int[] numbers = new int[]{1, 2, 3, 4, 5};
+
+    public String defineMood() throws NumberOutOfRangeException {
+        String statement = null;
+        System.out.println("Enter the digit");
+        Scanner sc = new Scanner(System.in);
+        int number = sc.nextInt();
+        if (number>7 || number<1) {
+            throw new NumberOutOfRangeException("Numbers is out of required range");
+        }
+        for (Integer day : numbers) {
```

Comment:
please remove the loop and array `numbers` - you can just check the value of a variable `number`